Sources/SwiftDocC/Infrastructure/Symbol Graph/SymbolReference.swift

```diff
@@ -93,3 +148,14 @@ private extension Array where Element == String {
         ).joined(separator: "_")
     }
 }
+
+extension UnifiedSymbolGraph.Symbol {
+    var sourceLanguages: Set<SourceLanguage> {
+        // FIXME: create a SymbolKit API to get all the languages of the symbol
```

Comment:
Can we refactor this with https://github.com/apple/swift-docc/pull/47/files#diff-c452db10128fc43b2e77212a1a282fe0834e541e10852fa8dd50f7c9b480b4e7R1104?

Reply:
Hm- what do you mean? 

```swift
moduleInterfaceLanguages = Set(
    unifiedSymbolGraph.symbols.flatMap(\.value.sourceLanguages)
)
```

That line is already calling this property.